src/Components/Aspire.Microsoft.Azure.Cosmos/PublicAPI.Unshipped.txt

```diff
@@ -10,5 +10,5 @@ Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.DisableTracing.get
 Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.DisableTracing.set -> void
 Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.MicrosoftAzureCosmosDBSettings() -> void
```

Comment:
```suggestion
Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.MicrosoftAzureCosmosSettings() -> void
```

If we are dropping `DB` in this library, we should do it for all public APIs in this library.

Reply:
Thanks! This is now covered with the latest commit.